src/style.css

```
@@ -1,3 +1,85 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+
+  --moyo-blue: #00acdc;
```

Create a variable with `:root` selector
```suggestion
  --main-color: #00acdc;
```